hack/jenkins/e2e.sh

```diff
@@ -645,6 +645,20 @@ case ${JOB_NAME} in
     : ${USE_KUBEMARK:="true"}
     : ${NUM_MINIONS:="10"}
     ;;
+
+  kubernetes-kubemark-gce-scale)
+    : ${E2E_CLUSTER_NAME:="kubernetes-kubemark-scale"}
+    : ${E2E_NETWORK:="kubernetes-kubemark-scale"}
+    : ${PROJECT:="kubernetes-scale"}
+    : ${E2E_UP:="true"}
+    : ${E2E_DOWN:="true"}
+    : ${E2E_TEST:="false"}
+    : ${USE_KUBEMARK:="true"}
+    : ${NUM_MINIONS:="40"}
+    : ${MASTER_SIZE:="n1-standard-32"}  # Note: not available in all zones
+    # Override GCE defaults.
```

Comment:
we don't set any GCE defaults in this script anymore, so this comment and the explicit override are probably no longer needed.